pkg/standalone/standalone.go

```diff
@@ -612,41 +612,45 @@ func untar(filepath, targetDir, binaryFilePrefix string) (string, error) {
 
 	tr := tar.NewReader(gzr)
 
+	foundBinary := ""
```

Comment:
no need for foundBinary ... can be refactored

Reply:
I added foundBinary to ensure that all files in the tarball are extracted, even if the reader finds the binary first. If I returned as soon as I found the binary, then the function may not extract everything from the tarball.